Posts/2022/07/ReadCMStatusMessages.md

```diff
@@ -1,375 +0,0 @@
----
-post_title: Reading Configuration Manager Status Messages With PowerShell
-username: francisconabas
-categories: PowerShell
-tags: SCCM, MECM, Status Message, Config Manager
-summary: This post's intent is to show how to read Configuration Manager status messages using WMI and Win32 API function FormatMessage.
----
```

Comment:
This PR is deleting your previous post. Please undo the delete. This previous post should not be part of this PR.